tox.ini

```diff
@@ -1,40 +1,73 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
-# Note: if you add new Python versions, please add them to
-# [testenv:coverage] depends as well!
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    coverage run -p -m zope.testrunner --test-path=src []
+    zope-testrunner --test-path=src []
 extras =
     test
-deps =
-    coverage
-setenv =
-     ZOPE_INTERFACE_STRICT_IRO=1
-# On certain platforms, at least OS X, pypy3 does not correctly read
-# a LANG environment variable; when cpython3.2 would return 'utf-8'
-# from locale.getpreferredencoding(), pypy3 returns 'us-ascii', causing
-# two test failures. Fix that by setting something it does read.
-     pypy3: LC_CTYPE=en_US.UTF-8
 
-[testenv:coverage]
+[testenv:lint]
+basepython = python3
 skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
 commands =
-    coverage erase
-    coverage combine
-    coverage html -i
-    coverage report --fail-under=93 --show-missing
-# parallel mode: make sure all builds complete before we run this one
-depends =
-    py27,py35,py36,py37,py38,pypy,pypy3
-parallel_show_output = true
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
 
 [testenv:docs]
-basepython =
-    python3
-commands =
-    sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
+basepython = python3
 extras =
     docs
+commands =
+    sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
+
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report -m --fail-under=90
```

Comment:
A 3% regression, eh?  Is that because we used to combine Python 2 and Python 3 coverage, and now we measure just Python 3?

I suppose that's acceptable.

(I wonder if we ought to change our existing `# Python 2`/`# Python 3` comments in `_compat.py` to `# pragma: PY2`/`# pragma: PY3` so coverage-python-version will recognize them, and then see what that does to the coverage numbers.)

Reply:
I misremembered -- we do not have `# Python 2` comments in _compat.py, just a `if PYTHON2:`.